scripts/buildtable.pl

```diff
@@ -182,13 +206,18 @@
 			die "Unknown field $field in $fn";
 		}
 		++$found{$field};
+		push @field_order, $field if $field_order[-1] ne $field;
```

Comment:
Running `./scripts/buildtable.pl` on this branch, I am seeing the following messages on each iteration that were not present on master:

`Use of uninitialized value in string ne at ./scripts/buildtable.pl line 209, <$F> line 2.`

Reply:
```
$ ./scripts/buildtable.pl
Use of uninitialized value in string ne at ./scripts/buildtable.pl line 209, <$F> line 2.
|- style="background-color: #ffcfcf"
| [[bip-0001.mediawiki|1]]
|
| BIP Purpose and Guidelines
| Amir Taaki
| Process
| Replaced
Use of uninitialized value in string ne at ./scripts/buildtable.pl line 209, <$F> line 2.
|- style="background-color: #cfffcf"
| [[bip-0002.mediawiki|2]]
|
| BIP process, revised
| Luke Dashjr
| Process
| Active
Use of uninitialized value in string ne at ./scripts/buildtable.pl line 209, <$F> line 2.
|- style="background-color: #ffffcf"
| [[bip-0003.md|3]]
|
| Updated BIP Process
| Murch
| Process
| Proposed
Use of uninitialized value in string ne at ./scripts/buildtable.pl line 209, <$F> line 2.
```
.../...

Reply:
Thanks -- rebased the fixup and merged the pull manually.